### PR TITLE
Implement testcase-Export001

### PR DESCRIPTION
**Pull request by cherrycl**
Implement #61  integration test case for  Export001 - Export events/readings to HTTP Server.

Signed-off-by: Cherry Wang <cherry@iotechsys.com>

### CODE REVIEW

File: TAF/utils/scripts/docker/docker-compose-end-to-end.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      edgex_profile: http-export
+      Service_Host: app-service-http-export
+      Clients_CoreData_Host: edgex-core-data
+      Clients_Logging_Host: edgex-support-logging
+      MessageBus_SubscribeHost_Host: edgex-core-data
+      Database_Host: edgex-mongo
+      Database_Username: appservice
+      Database_Password: password
+      EDGEX_SECURITY_SECRET_STORE: "${SECURITY_ENABLED}"
+      Writable_Pipeline_Functions_HTTPPostJSON_Parameters_url: http://172.17.0.1:8080
```
Comment (lenny-intel) [inline TAF/utils/scripts/docker/docker-compose-end-to-end.yaml:21]: Will this IP address always be `172.17.0.1`?

Comment (cherrycl) [reply TAF/utils/scripts/docker/docker-compose-end-to-end.yaml:21]: @lenny-intel As far as I understand, if using default bridge network on docker, we can always connect to docker host by 172.17.0.1 from container.

Comment (cloudxxx8) [reply TAF/utils/scripts/docker/docker-compose-end-to-end.yaml:21]: please see whether `host.docker.internal` can replace with the hard code ip address
https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds

Comment (cloudxxx8) [reply TAF/utils/scripts/docker/docker-compose-end-to-end.yaml:21]: don't use 8080 port because it is easy to be conflicted

Comment (cherrycl) [reply TAF/utils/scripts/docker/docker-compose-end-to-end.yaml]: @cloudxxx8  Update the http server port to 7770.
But using `host.docker.internal` to instead `172.17.0.1` occurred the following error when posting data.
`level=ERROR ts=2020-04-15T08:09:25.239280982Z app=AppService-http-export source=runtime.go:181 error="Post http://gateway.docker.internal:7770: dial tcp: lookup gateway.docker.internal on 127.0.0.11:53: no such host" correlation-id=6814a721-ec86-4c7b-8eb0-ba9d5aea1964 msg="Pipeline function #2 resulted in error"`

Comment (cherrycl) [reply TAF/utils/scripts/docker/docker-compose-end-to-end.yaml]: I tried the IP `172.17.0.1` on sandbox, it worked on both env., x86_64 and arm64.

File: TAF/utils/scripts/docker/arm64.env [hunk 2]
```diff
@@ -1,5 +1,10 @@
+# Image Variables
 virtual=nexus3.edgexfoundry.org:10004/docker-device-virtual-go-arm64:master
 mqtt=nexus3.edgexfoundry.org:10004/docker-device-mqtt-go-arm64:master
 random=nexus3.edgexfoundry.org:10004/docker-device-random-go-arm64:master
 modbus=nexus3.edgexfoundry.org:10004/docker-device-modbus-go-arm64:master
 modbusSimulator=panterdsd/diagslave
+appService=nexus3.edgexfoundry.org:10004/docker-app-service-configurable-arm64:master
+
+# Additional Variables
+DOCKER_HOST_IP=172.17.0.1
```
Comment (cloudxxx8) [inline TAF/utils/scripts/docker/arm64.env:10]: if the environment variable is common, please move to a single file, such as "common.env"

Comment (cherrycl) [reply TAF/utils/scripts/docker/arm64.env]: Move DOCKER_HOST_IP to common.env file.

File: TAF/utils/scripts/docker/docker-compose-end-to-end.yaml (hunk 1 above)
Comment (cloudxxx8) [reply TAF/utils/scripts/docker/docker-compose-end-to-end.yaml]: @lenny-intel it is the default ip address of the host for docker, and the http server is running on the host.  We made it become configurable in the environment variable.  Is is ok with you?

Comment (lenny-intel) [reply TAF/utils/scripts/docker/docker-compose-end-to-end.yaml]: Yes. Perfect! THX

**Comment by cherrycl** [comment]
rebased.

**Pull request merged**